.github/workflows/cuda-ci.yml

```diff
@@ -37,10 +63,14 @@ jobs:
         run: |
           source "${HOME}/conda/etc/profile.d/conda.sh"
           conda activate sklearn
-          pip install --verbose --no-build-isolation --config-settings editable-verbose=true --editable .
+          pip install ~/dist/cibw-wheels/$(ls ~/dist/cibw-wheels)
+
       - name: Run array API tests
         run: |
           source "${HOME}/conda/etc/profile.d/conda.sh"
           conda activate sklearn
           python -c "import sklearn; sklearn.show_versions()"
-          SCIPY_ARRAY_API=1 pytest -k 'array_api'
+
+          SCIPY_ARRAY_API=1 pytest --pyargs sklearn -k 'array_api'
```

Comment:
For my education: do we need the `--pyargs sklearn` because we are not in a directory where scikit-learn has been checked out?

Reply:
Yup, this is running the tests that are baked into the wheel. The wheel was built with the checked out code, so it's the same tests as the PR or branch.

At this point, the only reason to check out scikit-learn in the GPU runner is to get the lock file.